### ISSUE TITLE
Types: Module '"@floating-ui/react-dom"' has no exported member 'VirtualElement'

### ISSUE THREAD

**Issue by elektronik2k5** [floating-ui/floating-ui#1600]
Hello and thanks for this package!

When using the latest version of `@floating-ui/react-dom-interactions` (0.10.0) in a strict TS codebase with `"skipLibCheck": false` setting, compilation fails with this error:
```bash
> tsc --noEmit

node_modules/@floating-ui/react-dom-interactions/src/index.d.ts:2:33 - error TS2305: Module '"@floating-ui/react-dom"' has no exported member 'VirtualElement'.

2 import { ComputePositionReturn, VirtualElement } from '@floating-ui/react-dom';
                                  ~~~~~~~~~~~~~~


Found 1 error.
```

~I'll open a PR in a bit to fix it.~
PR with fix: https://github.com/floating-ui/floating-ui/pull/1601

**Comment by elektronik2k5** [comment]
Here's a `patch-package` file to work around the issue until an updated package is published:
```diff
diff --git a/node_modules/@floating-ui/react-dom/src/types.d.ts b/node_modules/@floating-ui/react-dom/src/types.d.ts
index f7f7d65..052338c 100644
--- a/node_modules/@floating-ui/react-dom/src/types.d.ts
+++ b/node_modules/@floating-ui/react-dom/src/types.d.ts
@@ -1,4 +1,4 @@
 export { useFloating } from './';
 export { autoPlacement, flip, hide, offset, shift, limitShift, size, inline, detectOverflow, getOverflowAncestors, autoUpdate, } from '@floating-ui/dom';
-export type { Platform, Placement, Strategy, Middleware, Alignment, Side, AlignedPlacement, Axis, Length, Coords, SideObject, Dimensions, Rect, ElementRects, ElementContext, ClientRectObject, Padding, RootBoundary, MiddlewareArguments, MiddlewareReturn, MiddlewareData, ComputePositionConfig, ComputePositionReturn, } from '@floating-ui/dom';
+export type { Platform, Placement, Strategy, Middleware, Alignment, Side, AlignedPlacement, Axis, Length, Coords, SideObject, Dimensions, Rect, ElementRects, ElementContext, ClientRectObject, Padding, RootBoundary, MiddlewareArguments, MiddlewareReturn, MiddlewareData, ComputePositionConfig, ComputePositionReturn, VirtualElement } from '@floating-ui/dom';
 export { arrow } from './';
```

Save it as `patches/@floating-ui+react-dom+0.6.0.patch` (after installing and configuring patch-package, of course).

**Comment by atomiks** [comment]
The RollingVersions action is currently running so should be released very soon :P (unless that flaky functional test fails lol)

**Comment by atomiks** [comment]
@elektronik2k5 v0.10.0 version is also a mistake. It should've been `0.0.10` 🤦 . I've deprecated that and the latest version is now 0.0.12 which I recommend upgrading to.

**Comment by elektronik2k5** [comment]
Thanks @atomiks! Confirmed to work in `"@floating-ui/react-dom-interactions": "0.0.12"`, although the package.json in it still points at the broken [`"@floating-ui/react-dom": "^0.6.0"`](https://github.com/floating-ui/floating-ui/blob/ee5a111f5403a0a245e4b4f4ea59322b84a62fec/packages/react-dom-interactions/package.json#L58) version instead of the newly released `0.6.1`. I suggest bumping that too.

**Comment by atomiks** [comment]
Upon removing the package/reinstalling (or a fresh install) it will use the latest transitive dep
